lang/cs/Org.Apache.REEF.Network/Group/Task/Impl/CommunicationGroupNetworkObserver.cs

```diff
@@ -30,8 +30,7 @@ namespace Org.Apache.REEF.Network.Group.Task.Impl
     /// Handles incoming messages sent to this Communication Group.
     /// Writable version
     /// </summary>
-    // TODO: Need to remove Iwritable and use IstreamingCodec. Please see Jira REEF-295.
-    public sealed class CommunicationGroupNetworkObserver : ICommunicationGroupNetworkObserver
+    internal sealed class CommunicationGroupNetworkObserver : ICommunicationGroupNetworkObserver
```

Comment:
since both interface and the class both are internal, I wouldassume your would like the method Register() also internal. It is public now.